python/README.md

```diff
@@ -4,3 +4,11 @@
 `.python-version` holds the recommended version of python that the repo was developed in.
 
 It is recommended to us `pyenv` when developing on this repo.
+
+# Development
+It is recommended to use `pyenv` and `vitualenv` when setting up a development environment for this repo.
+
+As such:
```

Comment:
Nit: may be nice to include a point saying that the python version for `pyenv` will be picked up automatically by the `.python-version` file?

Reply:
https://github.com/benjamincerigo/GildedRose-Refactoring-Kata/pull/1/commits/9989aa7bdefc60cabce5d1cb616e7e7b8c75f45f